meta-mentor-staging/recipes-multimedia/mplayer/mpv_0.15.0.bb

```diff
@@ -5,6 +5,7 @@ DEPENDS = "zlib ffmpeg jpeg virtual/libx11 xsp libxv \
            libxscrnsaver libv4l libxinerama \
            libxkbcommon \
 "
+RDEPENDS_${PN} += "libpulse libgbm libdrm"
```

Comment:
libpulse isn't a build dependency? How are these library references being put in the app if they are not used during build time.